.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,20 @@
-This PR:
+### What is the reason for this PR?
 
-- [ ] Adds a new feature ...
-- [ ] Covered by tests
-- [ ] Fixes #nnnn
+<!-- Explain your goals for this PR -->
+
+- [ ] A new feature
+- [ ] Fixes issue #
+
+### Author's checklist
+
+- [ ] Follow the [Contribution Guide](https://github.com/FakerPHP/Faker/blob/main/.github/CONTRIBUTING.md)
+- [ ] New features and changes are [documented](https://github.com/FakerPHP/fakerphp)
+
+### Summary of changes
+
+<!-- Give a quick explanation about your code changes here -->
+
+### Review checklist
+
+- [ ] All checks have passed
+- [ ] Changes are approved by maintainer
```

Comment:
Is this really needed since nobody will mark this as checked and just merges it imo

Reply:
Even if you don't mark it, it's good to have a list of mental steps.